bitrise.yml

```diff
@@ -129,13 +128,24 @@ workflows:
             - scheme: $XCODEBUILD_SCHEME
             - xcodebuild_options: $XCODEBUILD_OPTIONS
 
-  deploy-hockeyapp:
+  install-appcenter:
```

Comment:
We are missing this step inside https://github.com/netguru/how-we-roll/blob/579c4f9c866f4cf8651983b83ad30d0174240cbd/Mobile/iOS/templates/bitrise.yml#L299
Should it be added there?

Reply:
Which exactly step fo you mean?

Reply:
install-appcenter is missing in how-we-roll bitrise.yml template. I'm wondering if we should update this template, if this is necessary step.

Reply:
Oh, you should use this bitrise.yml for now https://github.com/netguru/how-we-roll/pull/217
as it's not merged yet.